backend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "backend",
```

Comment:
More specific name please, something like `auction9-backend`. The same applies for service name in serverless.yml.

Reply:
Okay, I will rename it in next commit